run-env-and-tests.sh

```diff
@@ -95,5 +95,5 @@ echo -e "
 
 echo -e "E2E TESTS\n"
 
-time ./environments/$ENV_NAME/start.sh --es "$ELK_VERSION" --kbn "$ELK_VERSION" --ror-es 1.63.0-pre2 --ror-kbn 1.63.0-pre3 --dev
```

Comment:
IMO we shouldn't modify this file. It should be the same on master and develop. AFAIR, we can set ROR plugins versions [here](https://github.com/beshu-tech/readonlyrest-e2e-tests/blob/5cf92aeac0e21326633ac1970ee06c01d1cf4dbc/.github/workflows/trigger-e2e-tests.yml#L52)

Reply:
Good point, done ✔️